domains/alfonce.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "theewizardone"
+    },
+    "records": {
+        "CNAME": "https://theewizardone.github.io/Morara_alfonce-awscloudportfolio/"
```

CNAME doesnt allow any trailing slashes (/)
```suggestion
        "CNAME": "theewizardone.github.io"
```